avocado/core/job.py

```diff
@@ -514,8 +510,27 @@ def _run(self):
         if 'FAIL' in summary:
             self.exitcode |= exit_codes.AVOCADO_TESTS_FAIL
 
+        self._post_run()
```

Comment:
Well the only issue in this is, that the `_post_run` is not executed on exception, nor on OptionValidationError, which breaks the `Pre` and `Post` parity.

So I think the `self._post_run` should be executed on the previous `post` job hook (line 567). Or do I miss some important issue?